pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -1,64 +1,74 @@
 { lib
 , stdenv
+, botocore
 , buildPythonPackage
-, isPy27
-, fetchPypi
+, cryptography
+, cssselect
+, fetchFromGitHub
 , glibcLocales
+, installShellFiles
+, itemadapter
+, itemloaders
+, jmespath
+, lxml
+, parsel
+, pillow
+, protego
+, pydispatcher
+, pyopenssl
+, pytest-twisted
 , pytestCheckHook
+, pythonOlder
+, queuelib
+, service-identity
+, sybil
 , testfixtures
-, pillow
 , twisted
-, cryptography
 , w3lib
-, lxml
-, queuelib
-, pyopenssl
-, service-identity
-, parsel
-, pydispatcher
-, cssselect
 , zope_interface
-, protego
-, jmespath
-, sybil
-, pytest-twisted
-, botocore
-, itemadapter
-, itemloaders
 }:
 
 buildPythonPackage rec {
-  version = "2.4.1";
-  pname = "Scrapy";
+  pname = "scrapy";
+  # 2.5.0 require h2<4
```

Comment:
Would you mind checking whether we can apply https://github.com/scrapy/scrapy/commit/c5b1ee810167266fcd259f263dbfc0fe0204761a?

Reply:
Seems to work if combined with another patch.